docs/designers-developers/developers/tutorials/create-block/submitting-to-block-directory.md

```diff
@@ -0,0 +1,89 @@
+# Share your Block with the World
+
+So you've created an awesome block? Care to share?
+
+**Contents**:
+1. Help users understand your block
+2. Analyze your plugin
+3. Zip & Submit
+
+## Step 1: Help users understand your block
+
+Providing straightforward, easy to understand block information is important to the block directory and our end users.
+
+**Guidelines**:
+
+- Name your block based on what it does
+- Clearly describe your block
+- Add Keywords for all contexts
+- Choose the right category
+
+### Name your block based on what it does
```

Comment:
I think we should take a clearer position on block names (specifically names not titles). Especially the namespace part. Unsurprisingly there's not much consistency in the plugin directory.

At a quick glance, I see people using namespaces that are:
* The plugin slug (like `card-block/main`, `pdf-viewer-block/standard`)
* The block name (`blockasaurus/blockasaurus`)
* The developer's employer (`a8c/waves`)
* The author's nickname or initials (`cgb/block-importdoc-block`)

None of these are wrong because we haven't given anyone clear guidance as to what they should use. I don't have a strong opinion, but I do thing the first two dot points above are sub-optimal. Using a very generic suffix like `blah/main` or `blah/standard` seems unhelpful. So does repetition like `blah/blah`.

Maybe we should recommend developers pick a namespace to use across all of their blocks, like in the last two dot points?

Reply:
I am tempted to suggest the last 2 are suboptimal because company names and initials are more likely to be repeated where as the block's folder name must be unique. Are there any issues if we suggest the namespace must match the plugin directory folder?

Reply:
I definitely think we should avoid `cgb` or other starter-content prefixes, since those are much more likely to collide.

The plugin slug will be unique in order to be uploaded to the directory, so that makes sense as a namespace to me. But I also think company name or username are fine too, and it can be left to the company/person to make sure they're not uploading multiple blocks with the same name. Initials are probably more likely to collide between people though, so I'd say discourage that.

Reply:
Another we should consider is that there is a desire to surface the block's name:
https://github.com/WordPress/gutenberg/issues/16521

Reply:
Surfacing the block's name, to me, suggests that the unique plugin slug is the best route.

Reply:
So is this decided, we'll _require_ the plugin slug to be used as a prefix? Should that be added into the guidelines, or is it enough to add a check to the Block Checker?

Reply:
"Name your block based on what it does"

It would be good with some example use cases to give a more detailed approach to how someone should approach the naming of their block. Have more detailed guidelines to a naming scheme as you are talking about above here. 

The guidelines also need to show examples of naming, describing, keywords and choosing the right category.
That can mean use a Create a Block documentation they should follow in getting these things right. (At present I do not know if the Create a Block documentation gives this needed information.)

Bottom line is that we can not assume a person will understand but we need to give clear guidelines to how things should be done.